packages/playground/src/__tests__/api/annotationsApi.int.spec.ts

```diff
@@ -111,7 +109,14 @@ describe.skip('Annotations API', () => {
     expect(annotation.creatingApp).toEqual(partial.creatingApp);
     expect(annotation.creatingAppVersion).toEqual(partial.creatingAppVersion);
     expect(annotation.status).toEqual(partial.status);
-    expect(annotation.data).toEqual(partial.data);
+    expect(annotation).toHaveProperty('data');
+    const annotation_data: any = annotation.data;
```

Comment:
```suggestion
    const annotationData: any = annotation.data;
```
🐫

Reply:
I think that makes sense to change this variable's name to cammelCase name. Can you change it?

Reply:
Done here: https://github.com/cognitedata/cognite-sdk-js/pull/771/commits/59592f87a2f7e43c0867d9409ae8cfe14f1174bb